src/tiles/item-image.ts

```diff
@@ -19,30 +19,34 @@ export class ItemImage extends LitElement {
 
   @property({ type: Boolean }) isListTile = false;
 
-  @state() private isDeemphasize: boolean = false;
+  @property({ type: Boolean }) isCompactTile = false;
+
+  @state() private isDeemphasize = false;
 
-  @state() private isWaveform: boolean = false;
+  @state() private isWaveform = false;
 
   @query('.item-image') private itemImageWaveform!: HTMLImageElement;
 
   protected updated(changed: PropertyValues): void {
     if (changed.has('model')) {
-      this.setDeemphsize();
+      this.setDeemphasize();
     }
   }
 
-  private setDeemphsize() {
+  // Don't deemphasize if item is a collection
+  private setDeemphasize() {
     this.isDeemphasize =
-      this.model?.collections.includes('deemphasize') ?? false;
+      (this.model?.mediatype !== 'collection' &&
```

Comment:
Q: do you need this part of the check `this.model?.mediatype !== 'collection'`?

Reply:
Yes